README.md

```diff
@@ -3,4 +3,8 @@ Juntos is a web application that lets you co watch streaming service such as you
 
 # Artefacts  
 * [Product Requirement Document](/prd/Juntos_PRD.pdf) 
-* [System Design](/system-design/Juntos_system_design.pdf)
\ No newline at end of file
+* [System Design](/system-design/Juntos_system_design.pdf)
+
+# UI Design
```

Comment:
I think this can be a section under "Artefacts"

Reply:
This is done. 
Cannot add links under pdf, will have to edit them separately to add the link, or will have to modify the screen to add a URL on top. Added Figma link in README instead. 
Created artefacts folder, added a Figma folder and page-name-folder under it.

Reply:
We can go with the following approach

```md
### UI design
 * Landing page [PDF]() [Figma]()
```

Reply:
@chirgjn  Now that all designs are complete. 
Do you want me to make any change to this?